libs/core/src/lib/message-box/message-box-header/message-box-header.component.html

```diff
@@ -1,4 +1,4 @@
-<header
+<div
```

Comment:
Why did you change it from header to div?

Reply:
To solve this I tried to use the ariaLabelledBy MessageBoxService option, but it didn't work, then I try to figure out what was wrong and the difference between the message field and the accessibility dialog implementation, because it is almost the same, then I found out, that the dialog uses the div instead of header and I changed it and it started working.

how to explain this magic, unfortunately, I don't know yet